app/src/main/java/grinnell/appdev/edu/lyles/MainActivity.java

```diff
@@ -20,38 +23,57 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
 
+        //Set toolbar as the action bar
         Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
         setSupportActionBar(toolbar);
 
         final DrawerLayout drawer = (DrawerLayout) findViewById(R.id.drawer);
 
+        //Enable navigation drawer buttons
```

Comment:
As we try to make the code more modular, let's extract all the configurations for the NavigationDrawer and make it a method on its own. You can simply highlight all the code related to the navdrawer, right-click and go to extract --> method. You can then call the method in onCreate(). This will keep onCreate() tidy and pretty short !